fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java

```diff
@@ -708,7 +708,6 @@ protected void getInfo(List<List<Comparable>> infos) {
         info.add(baseIndexName);
         info.add(rollupIndexName);
         info.add(rollupIndexId);
-        info.add(Integer.toString(rollupSchemaHash));
```

Comment:
daily test will failed because of the position changed.

Reply:
daily test case has been updated